### PR TITLE
Update some components to newer series to match Noble

### PR DESCRIPTION
**Pull request by seb128**
I didn't test build yet but still would welcome feedback on the proposed changes. Once merged we will probably also want to do a round of tag refresh for stable serie updates

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -269,11 +268,11 @@ parts:
   at-spi2-core:
     after: [ atk, meson-deps ]
     source: https://gitlab.gnome.org/GNOME/at-spi2-core.git
-    source-tag: 'AT_SPI2_CORE_2_44_1'
+    source-tag: '2.50.2'
```
Comment (sergio-costas) [inline snapcraft.yaml]: Mmm... are you sure about this?
imagen

Comment (sergio-costas) [reply snapcraft.yaml]: imagen

**Review by sergio-costas — CHANGES_REQUESTED** [review]
Just one detail.

File: snapcraft.yaml (hunk 1 above)
Comment (seb128) [reply snapcraft.yaml]: Thanks, I got confused by the webUI of /tags. I've changed at-spi back to the right tag format and forced pushed

**Review by sergio-costas — APPROVED** [review]

**Review by kenvandine — APPROVED** [review]

**Comment by seb128** [comment]
I've pushed another iteration with extra tweaks, some notes

- gee hasn't changed much in a while and the noble version is the same so switch the stage the deb instead of building
- removing at-spi2-atk which has been merged in at-spi2-core
- atk needs libgirepository1.0-dev for Glib-2.0.gir

The parts build locally but I've been hitting 'Failed to stage: parts list the same file with different contents or permissions' errors on parts that didn't change so I think it might be a local env problem

**Review by kenvandine — APPROVED** [review]
This looks good to me, let's merge it and get an LP build into edge

**Pull request merged**